Android App/app/src/main/java/org/cfp/citizenconnect/SplashScreen.java

```diff
@@ -65,7 +67,7 @@ public DownloadFilesTask(Context context) {
 
         @Override
         protected void onPreExecute() {
-            mSnakbar("Please wait...", null, 5000, 1, findViewById(R.id.coordinator), null);
+            Toast.makeText(SplashScreen.this,"Please wait ...",Toast.LENGTH_LONG).show();
```

Comment:
I also touched this part of code and introduced a new Snackbar because there was snackbar here which is now replaced with Toast. Is it a product decision to replace it and is it final?

Reply:
yes snack bar covered our logos